stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -976,6 +977,18 @@ extension NSDictionary : Sequence {
     }
   }
 
+  // Bridging subscript.
+  @objc
+  public subscript(key: Any) -> Any? {
```

Comment:
That's the only way we get overrideable methods in extensions right now. Without the custom selectors this would collide with the subscript methods from Cocoa.